src/main/java/com/contrastsecurity/maven/plugin/InstallAgentContrastMavenMojo.java

```diff
@@ -74,6 +93,20 @@ public String computeAppVersion(Date currentDate) {
         return computedAppVersion;
     }
 
+    public String computeSessionMetadata() {
+        List<String> metadata = new ArrayList<String>();
+
+        for(Map.Entry<String, String> entry: environmentToSessionMetadata.entrySet()) {
+            String environmentValue = System.getenv(entry.getKey());
+
+            if (environmentValue != null) {
+                metadata.add(String.format("%s=%s", entry.getValue(), environmentValue));
+            }
+        }
+
+        return StringUtils.join(metadata, ", ");
```

Comment:
Do we want spaces after the commas? I forget what the spec said.

Reply:
I _think_ spaces are fine by the RFC the docs mention, but all the docs examples don't have spaces after the comma, so I'm okay to remove it...